project/SparkRedshiftBuild.scala

```diff
@@ -175,10 +168,10 @@ object SparkRedshiftBuild extends Build {
         commitReleaseVersion,
         tagRelease,
         publishArtifacts,
-        releaseStepTask(spPublish),
         setNextVersion,
         commitNextVersion,
-        pushChanges
+        pushChanges,
+        releaseStepTask(spPublish)
```

Comment:
You can get rid of this last line.  Its broken now and will never work at this place in the release (we have already change back to a SNAPSHOT version).